tests/phpunit/tests/post/wpInsertPost.php

```diff
@@ -634,23 +634,6 @@ public function test_get_post_ancestors_within_loop() {
 		$this->assertSame( array( $parent_id ), get_post_ancestors( 0 ) );
 	}
 
-	/**
```

Comment:
I suggest keeping this here for the `wp_insert_post()` assertions, removing the `wp_update_post()` assertions in favor of the new file.

While not strictly related to this ticket, it will be handy for upcoming changes to phpunit https://core.trac.wordpress.org/ticket/62004#Problem4:coversattributesareonlysupportedatclasslevelx2